<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QSplitter" name="MainSplitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QFrame" name="frame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10000</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_Project_As"/>
    <addaction name="separator"/>
    <addaction name="actionExport_as"/>
    <addaction name="actionExport_image_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionNoise_Filling"/>
    <addaction name="actionPeak_Filtering"/>
    <addaction name="actionQuality_Assurance"/>
    <addaction name="actionNormalisation"/>
    <addaction name="actionMissing_Values_Imputation"/>
    <addaction name="actionTransformation"/>
    <addaction name="actionScaling"/>
    <addaction name="separator"/>
    <addaction name="actionComplete"/>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>Statistics</string>
    </property>
    <widget class="QMenu" name="menuUnivariate">
     <property name="title">
      <string>Univariate</string>
     </property>
     <widget class="QMenu" name="menuParametric">
      <property name="title">
       <string>Parametric</string>
      </property>
      <addaction name="actionT_test"/>
      <addaction name="actionANOVA_one_way"/>
     </widget>
     <widget class="QMenu" name="menuNon_parametric">
      <property name="title">
       <string>Non-parametric</string>
      </property>
      <addaction name="actionMann_Whitney_U_test"/>
      <addaction name="actionKruskal_Wallis"/>
     </widget>
     <addaction name="menuParametric"/>
     <addaction name="menuNon_parametric"/>
    </widget>
    <widget class="QMenu" name="menuMultivariate">
     <property name="title">
      <string>Multivariate</string>
     </property>
     <addaction name="actionPCA_1"/>
     <addaction name="actionPLS_DA"/>
    </widget>
    <addaction name="menuUnivariate"/>
    <addaction name="menuMultivariate"/>
    <addaction name="separator"/>
    <addaction name="actionNormality_Assessment"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_current_project"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuProcess"/>
   <addaction name="menuStatistics"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project...</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project...</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_Project_As">
   <property name="text">
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="actionExport_as">
   <property name="text">
    <string>Export as...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNoise_Filling">
   <property name="text">
    <string>Noise Filling</string>
   </property>
  </action>
  <action name="actionPeak_Filtering">
   <property name="text">
    <string>Peak Filtering</string>
   </property>
  </action>
  <action name="actionQuality_Assurance">
   <property name="text">
    <string>Quality Assurance</string>
   </property>
  </action>
  <action name="actionNormalisation">
   <property name="text">
    <string>Normalisation</string>
   </property>
  </action>
  <action name="actionMissing_Values_Imputation">
   <property name="text">
    <string>Missing Values Imputation</string>
   </property>
  </action>
  <action name="actionTransformation">
   <property name="text">
    <string>Transformation</string>
   </property>
  </action>
  <action name="actionScaling">
   <property name="text">
    <string>Scaling</string>
   </property>
  </action>
  <action name="actionComplete">
   <property name="text">
    <string>Complete</string>
   </property>
  </action>
  <action name="actionNormality_Assessment">
   <property name="text">
    <string>Normality Assessment</string>
   </property>
  </action>
  <action name="actionExport_image_as">
   <property name="text">
    <string>Export image as...</string>
   </property>
  </action>
  <action name="actionT_test">
   <property name="text">
    <string>t-test (paired/unpaired)</string>
   </property>
  </action>
  <action name="actionT_test_paired">
   <property name="text">
    <string>t-test (paired)</string>
   </property>
  </action>
  <action name="actionANOVA_one_way">
   <property name="text">
    <string>ANOVA (one-way)</string>
   </property>
  </action>
  <action name="actionMann_Whitney_U_test_unpaired">
   <property name="text">
    <string>Mann Whitney-U test (unpaired)</string>
   </property>
  </action>
  <action name="actionMann_Whitney_U_test">
   <property name="text">
    <string>Mann Whitney-U test (paired/unpaired)</string>
   </property>
  </action>
  <action name="actionKruskal_Wallis">
   <property name="text">
    <string>Kruskal-Wallis</string>
   </property>
  </action>
  <action name="actionPLS_DA">
   <property name="text">
    <string>PLS-DA</string>
   </property>
  </action>
  <action name="actionShow_current_project">
   <property name="text">
    <string>Show current project</string>
   </property>
  </action>
  <action name="actionScore_plot">
   <property name="text">
    <string>Score plot</string>
   </property>
  </action>
  <action name="actionLoading_plot">
   <property name="text">
    <string>Loading plot</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPCA_1">
   <property name="text">
    <string>PCA</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
